templates/snippets/head_meta.html

```diff
@@ -21,9 +21,11 @@
 {% if page.homepage %} {# project citation on home page #}
 <meta name="citation_title" content="Shakespeare and Company Project"/>
 <meta name="citation_date" content="2019"/>
-{% elif page.contentpage %} {# content page citation details #}
+{% elif page.contentpage or page.essaypage %} {# content/essay page citation details #}
 <meta name="citation_title" content="{{ page.title|escape }}"/>
-{# add multiple citation_author entries as needed, once we have them #}
+{% for author in page.authors %}
+<meta name="citation_author" content="{{ author.value }}">
```

Comment:
Author name should be in "lastname, first" format

Reply:
just to confirm: we didn't do this in PPA. happy to do it here, though

Reply:
I checked when I was trying to confirm the editor field, it should definitely be last name first. Should we make an issue for ppa?